src/caselawclient/models/judgments.py

```diff
@@ -3,8 +3,7 @@
 
 from requests_toolbelt.multipart import decoder
 
-from src.caselawclient.Client import MarklogicApiClient
-
+from ..Client import MarklogicApiClient
```

Comment:
If possible, I think it might be preferable to have the absolute import in case even if its just the ability to move this file without having to change the import in the future

would `from caselawclient.Client import MarklogicApiClient` work?